extensions/ui/org.eclipse.smarthome.ui.classic/src/main/java/org/eclipse/smarthome/ui/classic/internal/servlet/WebAppServlet.java

```diff
@@ -275,7 +289,9 @@ public boolean hasChangeOccurred() {
          */
         @Override
         public void stateUpdated(Item item, State state) {
-            // ignore if the state did not change
+            if (item instanceof GroupItem) {
+                changed = true;

```

Comment:
Standard groups can still have a [state](https://github.com/eclipse/smarthome/blob/master/bundles/core/org.eclipse.smarthome.core.test/src/test/groovy/org/eclipse/smarthome/core/items/GroupItemTest.groovy#L298). However, if they do not have a function their state is not derived from their member items.

If you change the code to only refresh the site for groups with a function you will miss state changes on those without function.